spec/apollo-federation/tracing_spec.rb

```diff
@@ -4,307 +4,320 @@
 require 'apollo-federation'
 
 RSpec.describe ApolloFederation::Tracing do
-  let(:base_schema) do
-    Class.new(GraphQL::Schema) do
-      use ApolloFederation::Tracing
+  RSpec.shared_examples 'a basic tracer' do
```

Comment:
Large diff to the spec mostly due to indentation and an edit on one of the line. I'll highlight where the change is